src/community/ogcapi/ogcapi-features/src/main/java/org/geoserver/ogcapi/v1/features/FeatureService.java

```diff
@@ -199,6 +199,12 @@ public WFSInfo getService() {
         return geoServer.getService(WFSInfo.class);
     }
 
+    @SuppressWarnings("unused")
+    public WFSInfo getServiceInfo() {
```

Comment:
Oh interesting... yeah, that callback uses reflection to access the info form the service object itself!
One day we should switch to just having an interface for it (ServiceInfoProvider or something similar).